homeassistant/components/obihai/connectivity.py

```diff
@@ -45,7 +45,7 @@ def __init__(
         self.host = host
         self.username = username
         self.password = password
-        self.serial: str
+        self.device_info: dict
```

Comment:
This seems not to be used on class level? and can be removed?

Reply:
I'm not sure I understand.  The attribute gets set in update(), and instance attributes should be declared in __init__.

Additionally, _if_ the user has multiple Obihai's, each entry will have a different ObihaiConnection and therefore have different values for device_info, so this shouldn't be set as a class attribute.